include/gatekeeper_acl.h

```diff
@@ -22,6 +22,12 @@
 #include "gatekeeper_config.h"
 #include "gatekeeper_net.h"
 
+/*
+ * Used by functions for matching particular packets, like BGP, ND packets.
+ * -NOMATCH means there is no match for a packet.
+ */
+#define NOMATCH	(2)
```

Comment:
If you agree with this change, don't forget to change the comments above `match_bgp()` and `match_nd()`.